README.md

```diff
@@ -1 +1,320 @@
+    messageRequest.sender = "admin@example.com";
+    messageRequest.subject = "Detected Text in Image at " + id;
+    messageRequest.messageBody = text;
+    messageRequest.contentType = gmail:TEXT_PLAIN;
+    _ = check gmailClient->sendMessage("me", messageRequest);
+    io:println("Email sent, image: ", id);
+}
+```
+
+NOTE: replace the `recipient` and `sender` values in the GMail message request properties accordingly.
+
+Now that we have the full Ballerina code, we can go to the next step in deploying it as an AWS Lambda function. 
+
+## Deployment
+
+- Building the Ballerina source:
+
+```bash
+$ ballerina build image-text-processor.bal 
+Compiling source
+    image-text-processor.bal
+Generating executable
+    image-text-processor.balx
+	@awslambda:Function: processImages
+
+	The Ballerina AWS Lambda layer information can be found at https://ballerina.io/deployment/aws-lambda.
+
+	Run the following commands to deploy each Ballerina AWS Lambda function:
+	aws lambda create-function --function-name <FUNCTION_NAME> --zip-file fileb://aws-ballerina-lambda-functions.zip --handler image-text-processor.<FUNCTION_NAME> --runtime provided --role <LAMBDA_ROLE_ARN> --timeout 10 --memory-size 1024
+	aws lambda update-function-configuration --function-name <FUNCTION_NAME> --layers <BALLERINA_LAYER_ARN>
+
+	Run the following command to re-deploy an updated Ballerina AWS Lambda function:
+	aws lambda update-function-code --function-name <FUNCTION_NAME> --zip-file fileb://aws-ballerina-lambda-functions.zip
+```
+
+The above command builds the Ballerina source, and the compiler, via the AWS Lambda extension, injects special functionality needed to execute it as an AWS Lambda function. The output of this is a file named `aws-ballerina-lambda-functions.zip`, which is the artifact that will be submitted to AWS Lambda. In the build process, it prints out the processed list of Lambda functions after the text "@awslambda:Function:"; Here, we only have the function `processImages`. These function names must be used in the placeholder `<FUNCTION_NAME>` in the given commands. Also, another placeholder that we will have to replace would be the `<LAMBDA_ROLE_ARN>`, which is the ARN of the earlier generated role `lambda-role`.
+
+Also, there is another placeholder named `<BALLERINA_LAYER_ARN>`, which is the ARN of the Ballerina runtime layer. There will be a Ballerina runtime layer for each Ballerina release, thus, the specific `<BALLERINA_LAYER_ARN>` should be looked up from the table available at [https://ballerina.io/deployment/aws-lambda](https://ballerina.io/deployment/aws-lambda).
+
+With the above placeholder values at hand, we can start to deploy the function we have developed. This consists of running the `aws lambda create-function` and the `aws lambda update-function-configuration` commands to finish the deployment. Below contains sample build commands for deploying the `processImages` function.
+
+- Deploying to AWS Lambda
```

Comment:
I think this will look better if we can make this a sub-title.

Reply:
Yes, true, thanks for the suggestion.